### PR TITLE
Missions - Hide blue target box overlay

### PR DESCRIPTION
**Pull request by Spoffy**
Also removes some old tracker code.

### CODE REVIEW

File: game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    Returns:
+        Marker name [STRING]
+
+    Example(s):
+        ["oscar8"] call vgm_g_fnc_loc_getTargetBoxBounds;
+ */
+
+params ["_targetBox"];
+
+format ["tbox_%1", _targetBox]
```
Comment (veteran29) [inline game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf:23]: There's already `vgm_g_fnc_missions_getZoneMarker`, maybe one should call the other?

File: game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        Gets the marker for a given target box
+
+    Parameter(s):
+        _targetBox - Name of the target box [STRING]
+
+    Returns:
+        Marker name [STRING]
+
+    Example(s):
+        ["oscar8"] call vgm_g_fnc_loc_getTargetBoxBounds;
```
Comment (veteran29) [inline game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf]:
```suggestion
        ["oscar8"] call vgm_g_fnc_loc_getTargetBoxMarker;
```

File: game/functions/systems/missions/client/fn_missions_coverMap.sqf [hunk 3]
```diff
@@ -82,3 +83,6 @@ _marker setMarkerDirLocal _dir;
 _marker setMarkerShapeLocal "rectangle";
 _marker setMarkerBrushLocal "border";
 _marker setMarkerColorLocal "colorblack";
+
+//--- Hide the target box overlay
+[_zone] call vgm_g_fnc_loc_getTargetBoxMarker setMarkerAlphaLocal 0;
```
Comment (veteran29) [inline game/functions/systems/missions/client/fn_missions_coverMap.sqf:88]: Not sure if we even need the function, there's already one for this.
```suggestion
_zone call vgm_g_fnc_missions_getZoneMarker setMarkerAlphaLocal 0;
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

File: game/functions/systems/missions/client/fn_missions_coverMap.sqf (hunk 3 above)
Comment (Spoffy) [reply game/functions/systems/missions/client/fn_missions_coverMap.sqf:88]: There wasn't in this branch when I was making it! 😆

File: game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf (hunk 1 above)
Comment (Spoffy) [reply game/functions/systems/locations/global/fn_loc_getTargetBoxMarker.sqf:23]: Didn't exist when this was made - updated missions_getZoneMarker to use it :)

**Review by veteran29 — APPROVED** [review]

**Pull request merged**